data/hai/midas.txt

```diff
@@ -0,0 +1,909 @@
+
+ship "Marauder Firebird" "Marauder Firebird (Bars)"
+	crew 37
+	outfits
+		"Plasma Cannon" 4
+		"Heavy Laser Turret"
+		"Heavy Anti-Missile Turret"
+		
+		"Boulder Reactor"
+		"Supercapacitor"
+		"S-970 Regenerator"
+		"Liquid Nitrogen Cooler"
+		"Bunk Room" 3
+		"Quantum Keystone"
+		"Outfits Expansion"
+		"Nerve Gas" 40
+		
+		"Ionic Afterburner"
+		`"Bondir" Atomic Steering`
+		"Hyperdrive"
+	cargo
+		outfits
+			"Gold Bars" 15
+
+# Ship of She of the Void, Shadow of Mintaka, leader of the Marauders.
+ship "Antlion"
+	sprite "ship/hai antlion"
+	thumbnail "thumbnail/hai antlion"
+	attributes
+		category "Heavy Warship"
+		cost 35200000
+		mass 520
+		bunks 67
+		"cargo space" 56
+		drag 6.9
+		"engine capacity" 180
+		"fuel capacity" 600
+		"heat dissipation" 0.66
+		hull 9000
+		"outfit space" 696
+		"required crew" 35
+		shields 24000
+		"weapon capacity" 280
+		weapon
+			"blast radius" 260
+			"shield damage" 3600
+			"hull damage" 1820
+			"hit force" 2500
+	outfits
+		"Ionic Blaster Prototype" 4
+		"Ionic Turret Prototype" 2
+		"Chameleon Anti-Missile" 2
+
+		"Fuel Processor"
+		Hyperdrive
+		"Hai Diamond Regenerator"
+		"Systems Core (Medium)"
+
+		"Geode Reactor"
+		"Hai Valley Batteries"
+		"Value Detector" 2
+
+		"Quantum Keystone"
+		"Ionic Blade"
+		"Thermal Repeater Rifle" 61
+		"Dual Pulse Pistols" 5
+
+		"Type 4 Radiant Thruster"
+		`"Bufaer" Atomic Steering`
+		`"Biroo" Reverse Thruster`
+	cargo
+		outfits
+			"Gold Bars" 50
+	"steering engine" 62 10
+		zoom 0.6
+		right
+		angle -115
+	"steering engine" -62 10
+		zoom 0.6
+		left
+		angle 115
+	"reverse engine" 0 -96
+		zoom 0.8
+	engine 44.5 64
+		zoom 0.8
+	engine -44.5 64
+		zoom 0.8
+	engine 0 110
+	gun -24 -100 "Ionic Blaster Prototype"
+	gun 24 -100 "Ionic Blaster Prototype"
+	gun -15 -97 "Ionic Blaster Prototype"
+	gun 15 -97 "Ionic Blaster Prototype"
+	turret -22.5 -57.5 "Chameleon Anti-Missile"
+	turret 22.5 -57.5 "Chameleon Anti-Missile"
+	turret -15 41 "Ionic Turret Prototype"
+	turret 15 41 "Ionic Turret Prototype"
+	bay Drone -67 -76
+		under
+	bay Drone 67 -76
+		under
+	bay Drone 0 70.5
+		over
+	leak leak 40 50
+	leak flame 50 80
+	leak "big leak" 60 50
+	explode "huge explosion" 10
+	explode "large explosion" 30
+	explode "medium explosion" 50
+	explode "small explosion" 40
+	explode "tiny explosion" 40
+	"final explode" "final explosion large"
+	description "This ship was designed to repel the raiding ships, near the end of the Korath raids. Whilst it would not be able to defeat one of their ships alone, and would struggle in a prolonged fight, its energy weapons and drone support allow it to hold its own until reinforcements arrive."
+	description "	The prototype was stolen after the raids meaning that this ship design was never produced again, given it would be of little use in the Unfettered conflict. Nowadays the Antlion is renowned for attacking vulnerable targets and disappearing before anyone can react, as if always one move ahead."
```

Comment:
```suggestion
	description "	The prototype was stolen after the raids and this ship design was never produced again, given it would be of little use in the Unfettered conflict. Nowadays the Antlion is renowned for attacking vulnerable targets and disappearing before anyone can react, as if always one move ahead."
```